src/installer/features/version/loader.ts

```diff
@@ -0,0 +1,69 @@
+/**
+ * Version file loader
+ * Manages the .nori-installed-version file lifecycle
+ */
+
+import { unlinkSync, existsSync } from "fs";
+
+import { info, success } from "@/installer/logger.js";
+import {
+  getCurrentPackageVersion,
+  saveInstalledVersion,
+  getVersionFilePath,
+} from "@/installer/version.js";
+
+import type { Config } from "@/installer/config.js";
+import type { Loader } from "@/installer/features/loaderRegistry.js";
+
+/**
+ * Install version file - save current package version
+ * @param args - Configuration arguments
+ * @param args.config - Runtime configuration
+ */
+const installVersion = async (args: { config: Config }): Promise<void> => {
+  const { config } = args;
+
+  const currentVersion = getCurrentPackageVersion();
+  if (currentVersion == null) {
+    info({
+      message: "Could not determine package version, skipping version file",
+    });
+    return;
+  }
+
+  saveInstalledVersion({
+    version: currentVersion,
+    installDir: config.installDir,
+  });
+
+  success({
+    message: `✓ Version file created: ${getVersionFilePath({ installDir: config.installDir })}`,
```

Comment:
remove the dependency on `getVersionFilePath`, you know exactly what the path is when you inline the saveInstalledVersion call